### PR TITLE
Implement new format for month/year short

### PR DESCRIPTION
**Pull request by Rantelo**
**example**
formatMonthYearShort(new Date(2018, 1, 2, 3, 4)); //=> 'Feb 18'

### CODE REVIEW

File: source/utilities/time/index.js [hunk 1]
```diff
@@ -163,6 +163,21 @@ const formatDayMonthYearShort = (date = new Date(), locale='mx') => {
...
+ * formatMonthYearShort(new Date(2018, 1, 2, 3, 4)); //=> 'Feb 18'
+ *
+ * @param {Date} [date] - Date. If not provided, `Date.now()`
+ * @param {string} [locale] - Locality string. `mx` by default
+ * @returns {string}
+ */
+const formatMonthYearShort = (date = new Date(), locale='mx') => {
+  let [year, month] = dateSpread(date, false);
+  return `${MONTHS_SHORT[locale][month-1]} ${year % 100}`;
+}
```
Comment (uquintana) [inline source/utilities/time/index.js]: Semicolon pana :)

**Review by madnessDann — APPROVED** [review]

File: source/utilities/time/index.js (hunk 1 above)
Comment (Rantelo) [reply source/utilities/time/index.js]: True that. I'll also include the semicolons for the rest of functions in this module.

**Review by Tcp93001 — COMMENTED** [review]
Nothing to add...

File: source/utilities/time/test.js [hunk 2]
```diff
@@ -170,6 +171,17 @@ describe("Testing time utilities", () => {
     });
   });
 
+  describe("formatMonthYearShort()", () => {
+    const {formatMonthYearShort} = util;
+    it("Generates a DD de MMM YY format", () => {
```
Comment (uquintana) [inline source/utilities/time/test.js]: This text is the same from `formatDayMonthYearShort`

**Review by Tcp93001 — APPROVED** [review]

File: source/utilities/time/test.js (hunk 2 above)
Comment (Rantelo) [reply source/utilities/time/test.js]: True. Now is fixed.
Thanks

**Pull request merged**